static/app/main.tsx

```diff
@@ -39,6 +40,9 @@ function Main() {
             {routes()}
           </Router>
         </OnboardingContextProvider>
+        {ConfigStore.get('devtools')?.reactQuery && (
+          <ReactQueryDevtools initialIsOpen={false} position="bottom-left" />
+        )}
```

Comment:
can we check the constant directly instead of ConfigStore

Reply:
We were chatting through this a little in a thread: https://sentry.slack.com/archives/C050ZU8FZ8C/p1711565322569459?thread_ts=1711146656.860459&cid=C050ZU8FZ8C 

I thought it was kinda nice to be able to use either dev env to set the flag and then read it -- that way it's setup for either the `sentry devserver` or `yarn dev-ui` workflows.

I don't have strong feelings though, i agree with ya that if you're using these devtools you'd likely be using the `yarn dev-ui` workflow too.